webhookClient/metrics.go

```diff
@@ -57,9 +60,32 @@ type Measures struct {
 	WebhookRegistrationOutcome metrics.Counter
 }
 
-//NewMeasures realizes desired metrics.
+// MeasuresIn is an uber/fx parameter with the webhook registration counter
+type MeasuresIn struct {
+	fx.In
+	WebhookRegistrationOutcome metrics.Counter `name:"webhook_registration"`
+}
+
+// NewMeasures realizes desired metrics.
 func NewMeasures(p provider.Provider) *Measures {
 	return &Measures{
 		WebhookRegistrationOutcome: p.NewCounter(WebhookRegistrationOutcome),
 	}
 }
+
+// NewProvideMeasures converts MeasuresIn to Measures
+func NewProvideMeasures(in MeasuresIn) *Measures {
```

Comment:
I'm not sure why this is needed?

Reply:
`NewPeriodicRegisterer` takes in a `Measures` struct, so we need a conversion from the `MeasuresIn` that is created by uber/fx to `Measures`. However, this conversion functionality can be moved into `ProvideMetrics` instead, and then we can get rid of `NewProvideMeasures`.